subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -229,3 +229,31 @@ Run with `--watch-fs` on the command line::
 This enables watching the file-system for this build only.
 Put `org.gradle.unsafe.watch-fs=true` in your `gradle.properties`::
 This enables watching the file-system for all builds, unless explicitly disabled with `--no-watch-fs`.
+
+[[sec:daemon_watch_fs_troubleshooting]]
+=== Troubleshooting file-system watching
+
+Limitations::
```

Comment:
Should we clarify that these limitations are stuff we are actively working on removing before the feature goes into production?

Reply:
I added a sentence.